wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationReadReceiptSettingChangedCellDescription.swift

```diff
@@ -30,26 +30,25 @@ struct ReadReceiptViewModel {
         return iconColor.map { icon.makeImage(size: .tiny, color: $0) }
     }
 
-    func createSystemMessage(template: String) -> NSAttributedString {
-        var updateText: NSAttributedString! = .none
+    func createSystemMessage(template: String) -> NSAttributedString? {
+        var updateText: NSAttributedString?
```

Comment:
Does this need to be optional if we always give it a value?

Reply:
Yes, it doesn't need to be optional in this implementation. 
Fixed